pages.es/netbsd/df.md

```diff
@@ -0,0 +1,32 @@
+# df
+
+> Muestra una visión general del uso del espacio en disco del sistema de archivos.
+> Más información: <https://man.netbsd.org/NetBSD-9.3/df.1>.
+
+- Muestra todos los sistemas de ficheros y su uso de disco usando unidades de 512 bytes:
+
+`df`
+
+- Utiliza unidades legibles por el ser [h]umano (basadas en potencias de 1024):
```

Comment:
IMO, the wording is a little bit off.

What do you think @vitorhcl?

```suggestion
- Utiliza palabras [h]umanas para indicar unidades (basadas en potencias de 1024):
```

Reply:
As the units are represented by a single letter, I think his description was better

Reply:
Alright, I'll stay as it is